src/app/books/book-details/book-details.component.html

```diff
@@ -0,0 +1,61 @@
+<div class="container">
+    <div class="card">
+        <div class="card-header">
+            <span>Book Details</span>
+        </div>
+        <div class="card-body">
+            <div class="row">
+                <div class="col-md-6">
+                    <div class="text-left">
+                        <button type="button" class="btn btn-primary" style="margin-right:30%"
+                            routerLink="/books">Back</button>
+                        <img class="mx-auto d-block" src = "{{ book.imageUrl }}" width="50%" height="80%"/>
+                    </div>
+                    <br><br>
+                </div>
+                <div class="col-md-6">
+                    <table class="table table-bordered">
+                        <tr>
+                            <td>Book Id</td>
+                            <td>{{ book.bookId }}</td>
+                        </tr>
+                        <tr>
+                            <td>Title</td>
+                            <td>{{ book. title }}</td>
```

Comment:
Any specific reason for space  <td>{{ book. title }}</td> ?

Reply:
That was a mistake. Solved